src/components/CharacterCard/CharacterCard.test.jsx

```diff
@@ -0,0 +1,38 @@
+import { render, screen } from '@testing-library/react';
+import CharacterCard from './CharacterCard';
+import { characterData, homer } from '../../services/characterData'
+import { rest } from 'msw'
+import { server } from '../../views/Main/Main.test'
+
+test('Should render a character information on card', () => {
+    server.use(
+        rest.get('https://thesimpsonsquoteapi.glitch.me/quotes?count=20', (req, res, ctx) => res(ctx.json([ characterData ])))
+      ) 
+    render(
+        <CharacterCard />
+    )
+        // for heading level 3 is h3 and level 4 is h4
+    const characterName = screen.getByRole('heading', {level: 3 });
+    expect(characterName).toBeInTheDocument();
+
+    const characterImage = screen.getByAltText(/characterimg/i);
+    expect(characterImage).toBeInTheDocument();
+
+    const characterQuote = screen.getByRole('heading', { level: 4 });
+    expect(characterQuote).toBeInTheDocument();
```

Comment:
Consider adding in details about the mock, such as character name, to be within the component test.